packages/docusaurus-theme-common/src/components/Details/styles.module.css

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/*
+CSS variables, meant to be overriden by final theme
+ */
+.details {
+  --docusaurus-details-summary-arrow-size: 0.38rem;
+  --docusaurus-details-transition: transform 200ms ease;
+  --docusaurus-details-decoration-color: grey;
+}
+
+.details > summary {
+  position: relative;
+  cursor: pointer;
+  list-style: none;
+  margin-left: 1.8rem;
+}
+
+.details > summary::-webkit-details-marker {
+  display: none;
+}
+
+.details > summary:before {
+  position: absolute;
+  top: 0.45rem;
+  left: -1.2rem;
+
+  /* CSS-only Arrow */
+  content: '';
+  width: 0;
+  height: 0;
+  border-top: var(--docusaurus-details-summary-arrow-size) solid transparent;
+  border-bottom: var(--docusaurus-details-summary-arrow-size) solid transparent;
+  border-left: var(--docusaurus-details-summary-arrow-size) solid
+    var(--docusaurus-details-decoration-color);
+
+  /* Arrow rotation anim */
+  transform: rotate(0deg);
+  transition: var(--docusaurus-details-transition);
+  transform-origin: calc(var(--docusaurus-details-summary-arrow-size) / 2) 50%;
+}
+
+/* When JS disabled/failed to load: we use the open property for arrow animation: */
+.details[open]:not(.isClient) > summary:before,
+/* When JS works: we use the data-attribute for arrow animation */
+.details[data-collapsed='false'].isClient > summary:before {
+  transform: rotate(90deg);
+}
+
+.collapsibleContent {
+  margin-top: 1rem;
+  border-top: 1px solid var(--docusaurus-details-decoration-color);
+  padding-top: 1rem;
```

Comment:
Technically, in this case we could use CSS utility classes (margin-top--md, padding-top--md) instead of writing according CSS code, but I'm not sure about that.

Reply:
I didn't do that because this comp is in theme-common and decoupled from Infima for later usage in other themes. Eventually, it could expose its own CSS variables to customize spacing but that seems overkill for now.